### ISSUE TITLE
F-Droid build not reproducible

### ISSUE THREAD

**Issue by linsui** [Staacks/alpharemote#9]
org/staacks/alpharemote/service/AlphaRemoteService$EntriesMappings.class is only in your apk. Could you please take a look? Thanks!

**Comment by Staacks** [comment]
Sorry, I am a bit confused here. Is this about 0.19 (190) that I uploaded yesterday? The F-Droid build process has not yet pulled this version, has it?

**Comment by Staacks** [comment]
Triggered a test build on gitlab and you are (of course) right.

I do not see any change in the code that could cause this and searching for that class seems to yield some internal Kotlin stuff for enums. So, at the moment I think that something must have changed at my local toolchain, but my apt logs tell me that the last update to Android Studio happened before I made the last successful reproducible build. I hope I find some time for more digging...

**Comment by Staacks** [comment]
Ok, I got it working again. I replaced the apk in the current github release (F-Droid hasn't downloaded it yet as far as I can tell, so it should have been in time) and the test pipeline on gitlab now successfully compares it to the built library.

I am not 100% sure what went wrong, but I think it was caused by using the "google-android-platform-tools-installer" package to install some tools that I needed for testing different versions of LineageOS on my Pixel 3. This package installs platform tools 35.0.0 instead of the 35.0.2 that was already installed by Android Studio and that is correctly used by F-Droid. I suspect that some file got replaced in the process unnoticed. In the end I purged and reinstalled Android Studio and got a build that matches.
